ods-devenv/scripts/deploy.sh

```diff
@@ -1310,8 +1310,9 @@ function create_configuration() {
     sed -i "s/PROV_APP_ATLASSIAN_DOMAIN=.*$/PROV_APP_ATLASSIAN_DOMAIN=${odsbox_domain}/" ods-core.env
     sed -i "s/PROV_APP_CROWD_PASSWORD=.*$/PROV_APP_CROWD_PASSWORD=ods/" ods-core.env
     sed -i "s|CROWD_URL=.*$|CROWD_URL=http://${atlassian_crowd_host}:${atlassian_crowd_port_internal}/crowd|" ods-core.env
-    sed -i "s/PROV_APP_CONFLUENCE_ADAPTER_ENABLED=.*$/PROV_APP_CONFLUENCE_ADAPTER_ENABLED=false/" ods-core.env
-    sed -i "s/PROV_APP_PROVISION_CLEANUP_INCOMPLETE_PROJECTS_ENABLED=.*$/PROV_APP_PROVISION_CLEANUP_INCOMPLETE_PROJECTS_ENABLED=true/" ods-core.env
+    sed -i "s|PROV_APP_CONFLUENCE_ADAPTER_ENABLED=.*$|PROV_APP_CONFLUENCE_ADAPTER_ENABLED=false|" ods-core.env
+    sed -i "s|PROV_APP_AUTH_BASIC_AUTH_ENABLED=.*$|PROV_APP_AUTH_BASIC_AUTH_ENABLED=true|" ods-core.env
+    sed -i "s|PROV_APP_PROVISION_CLEANUP_INCOMPLETE_PROJECTS_ENABLED=.*$|PROV_APP_PROVISION_CLEANUP_INCOMPLETE_PROJECTS_ENABLED=false|" ods-core.env
```

Comment:
@georgfedermann `PROV_APP_PROVISION_CLEANUP_INCOMPLETE_PROJECTS_ENABLED` should be set to `false` in `PROD` environments. For ods-devenv we could set it to `true`.